components/overflow-group/README.md

```diff
@@ -1,10 +1,40 @@
 # Overflow Groups
+The `d2l-overflow-group` element can be used to add responsiveness to a set of buttons, links or menus. 
 
-## d2l-overflow-group
-The `d2l-overflow-group` element can be used to add responsiveness to a set of buttons, links or menus. Items added to this container element will no longer wrap onto a second line when the container becomes to small but will be added too a dropdown menu with configurable styling.
+<!-- docs: demo size:medium autoSize:false display:block -->
```

Comment:
super nit: I've been trying to alphabetize the properties (other than if `name` is there I usually put that first)